api/src/main/java/org/openmrs/module/patientflags/api/FlagService.java

```diff
@@ -218,12 +273,31 @@
 	/**
 	 * Retrieve a specific Priority based on its priorityId
 	 * 
-	 * @param tagId
-	 * @return the Tag which matches the specific ID
+	 * @param priorityId
+	 * @return the Priority which matches the specific ID
 	 */
 	@Transactional(readOnly = true)
 	@Authorized(value = { "Manage Flags", "Test Flags" }, requireAll = false)
 	public Priority getPriority(Integer priorityId);

```

Comment:
You need to have at least a line between methods.

Reply:
Done